### PR TITLE
Split disk crio setup

### PR DESCRIPTION
**Pull request by kannon92**
Adding presubmits for the e2e tests for split filesystem.

### CODE REVIEW

**Comment by kannon92** [comment]
/hold
got one thing to fix

File: jobs/e2e_node/crio/templates/base/60-storage-split-disk.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[storage]
+
+# Default Storage Driver
+driver = "overlay"
+
+runroot = "/var/containers/storage"
+# Ephemeral Storage and writeable layer for containers
+graphroot = "/var/lib/containers"
+# Storage for images
+imagestore = "/tmp/containers"
```
Comment (haircommander) [inline jobs/e2e_node/crio/templates/base/60-storage-split-disk.conf:10]: nit: feels to me it'd be more accurate to have this be `/tmp/images`

**Pull request merged**